libraries/rush-lib/src/cli/actions/ChangeAction.ts

```diff
@@ -297,6 +306,12 @@ export class ChangeAction extends BaseRushAction {
     } catch (error) {
       throw new Error(`There was an error creating a change file: ${(error as Error).toString()}`);
     }
+    if (this._commitChangesFlagParameter.value) {
+      this._stageAndCommitGitChanges(
+        '*',
```

Comment:
It'd be better to specifically enumerate the changefiles that this command created, instead of committing every change to the `common/changes` folder.

Reply:
This could be done fairly easily by updating `_writeChangeFiles` to return the list of file paths that it created and then passing those in to `rush add` as arguments.

Reply:
This also raises a potential issue where this flag will commit anything that the user has staged, in addition to the changefiles. That's probably not the desired behavior.

Reply:
We can commit only specific staged files, not all staged files by passing them to the commit command, so it's fine. 
I added the changes to `_writeChangeFiles`. Also tested them manually by running `rush change`, appending a changefile, and then afterwards running `rush change -c`. Only the latter changefile was committed, which can be seen in one of the commits. I also added a warning if `rush change -c` is run, but no changefiles were made, to warn the user that no commits were made.

By `rush add` did you mean `git add` or am I missing something?

Reply:
Yeah I meant `git add`.